charts/xui-webapp/values.preview.template.yaml

```diff
@@ -4,6 +4,30 @@ nodejs:
   environment:
        IDAM_URL: https://idam-api.aat.platform.hmcts.net
        S2S_URL: http://rpe-service-auth-provider-aat.service.core-compute-aat.internal
+       APPINSIGHTS_INSTRUMENTATIONKEY: ${APPINSIGHTS_INSTRUMENTATIONKEY}
        S2S_SECRET: ${S2S_SECRET}
        IDAM_SECRET: ${IDAM_SECRET}
        NODE_TLS_REJECT_UNAUTHORIZED: '0'
+       # Health Endpoints
+       HEALTH_CCD_COMPONENT_API: http://ccd-api-gateway-web-aat.service.core-compute-aat.internal/health
+       HEALTH_CCD_DATA_API: http://ccd-data-store-api-aat.service.core-compute-aat.internal/health
+       HEALTH_DOCUMENTS_API: http://dm-store-aat.service.core-compute-aat.internal/health
+       HEALTH_EM_ANNO_API: http://em-anno-aat.service.core-compute-aat.internal/health
+       HEALTH_TERMS_AND_CONDITIONS_API: http://xui-terms-and-conditions-aat.service.core-compute-aat.internal/health
+       # Services CCD Endpoints
+       SERVICES_CCD_COMPONENT_API: http://ccd-api-gateway-web-aat.service.core-compute-aat.internal
```

Comment:
ideally we need to make these as 'templated' variables

Reply:
This is a clean-up task. But for a later day